conf/deployment/azure/ipi_1az_rhcos_3m_3w.yaml

```diff
@@ -7,7 +7,6 @@ ENV_DATA:
   deployment_type: 'ipi'
   region: 'eastus'
   azure_base_domain_resource_group_name: 'ocsqe'
-  base_domain: 'azure.qe.rh-ocs.com'
```

Comment:
Can you please document this somewhere that this value has to be passed by configuration file - that QE has those two files and one of those has to be provided by --ocsci-conf parameter?

I would add it for example here:
https://github.com/red-hat-storage/ocs-ci/blob/master/docs/supported_platforms.md#azure

where you can document that if someone outside of QE would like to use their own domain, they have to specify in custom config file.

It can be done in separate PR, so approving this one but please open the issue and document this somewhere.

Reply:
I see, that deployment on Azure is not documented at all :-( ... I'm not sure, if I'm the right one to start with the section, but I'll try to formulate at least some general note.

Reply:
> deployment on Azure is not documented at all

That file was created few months later after Azure deployment was implemented.

The main issue here in this pull request is that we are changing behavior, in a weird way which requires an explanation.

Reply:
> The main issue here in this pull request is that we are changing behavior, in a weird way which requires an explanation.

@mbukatov I understand and agree and I definitely count with creating and sending email once we will be ready with merging this and the related [MR!545](https://gitlab.cee.redhat.com/ocs/ocs4-jenkins/-/merge_requests/545) - which changes slightly also the workflow for deployment via Jenkins job. I'll be glad if you can help me then with the content of such email.

Reply:
Fixed by 455f856.